package.json

```diff
@@ -110,6 +110,7 @@
     "pretty-bytes": "2.0.1",
     "request": "2.73.0",
     "rimraf": "2.5.1",
+    "seedrandom": "2.4.3",
```

Comment:
If `shuffle-seed` works, use it here instead of `seedrandom`.

You'll also need to update `yarn.lock`. Your Travis build should contain an error message saying how to do so.

Reply:
done